### PR TITLE
adding 2 properties to oidc config 

### PR DESCRIPTION
**Pull request by samshuster**
### Summary of Changes

I modified one uri and added an additional uri that were required for my configuration (not sure if this is universal? if not I will just add them to readme)

Adding more info to the oidc_config.yaml readme. NOTE: I did not include information on how to build custom frontend at this time.

### Documentation

I added some information to the oidc README (is this the right place still?) for how to get specifically okta and amundsen integrated

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: docs/authentication/oidc.md [hunk 1]
```diff
@@ -97,3 +97,35 @@ AUTH_USER_METHOD = get_auth_user
...
+3. When you start up helm you will need to provide some properties. Here are the properties that need to be overridden for oidc to work:
+    ```yaml
+    oidcEnabled: true
+    createOidcSecret: true
+    OIDC_CLIENT_ID: YOUR_CLIENT_ID
+    OIDC_CLIENT_SECRET: YOUR_SECRET_ID
+    OIDC_ORG_URL: https://edmunds.okta.com
+    OIDC_AUTH_SERVER_ID: default
+    ```
+4. At present, there is no oidc build of the frontend. So you will need to build an oidc build yourself and upload it to, for example ECR, for use by k8s.
```
Comment (samshuster) [inline docs/authentication/oidc.md]: if it would be useful, I can provide more info on how to build frontend with oidc

Comment (feng-tao) [reply docs/authentication/oidc.md]: thanks. I think it would be helpful.

**Comment by samshuster** [comment]
ok added some more documentation. Hopefully its in the right place, although I would understand if it should be moved to amundsenfrontendlibrary.

**Comment by feng-tao** [comment]
I think eventually we should consolidate all the docs here and redirect other repos to here for reference.

**Pull request merged**